### ISSUE TITLE
Move into core codebase

### ISSUE THREAD

**Issue by Tyriar** [jerch/xterm-addon-image#49]
Finally got around to pulling this into vscode in https://github.com/microsoft/vscode/pull/182442, it works really well 👏

I've made it currently disabled by default and marked a setting with experimental. One of the things that would be great to do before enabling it by default is to merge it into the core codebase so we can strongly type the internal API usage https://github.com/microsoft/vscode/issues/182445. Thoughts?

**Comment by jerch** [comment]
Yes moving it over as official addon would remove some of the typing quirks it currently faces. :+1: 

Regarding the vscode integration - maybe its a good idea to mention, that images are currently not yet part of the restored terminal content (would avoid getting issue reports regarding this).

**Comment by Tyriar** [comment]
@jerch yep, called that out in the setting and https://github.com/microsoft/vscode/issues/182445. When I add does on the support I'll call it out there to reduce some reports until it works

**Comment by jerch** [comment]
@Tyriar FYI - did a last addon release from this repo for xtermjs 5.2, as it was more convenient for me at the current stage. Will move things over to the main repo in the next weeks, so prolly with the next minor of xtermjs the addon can be packed from there.
